doc/source/whatsnew/v0.25.2.rst

```diff
@@ -103,6 +103,7 @@ Other
 - Compatibility with Python 3.8 in :meth:`DataFrame.query` (:issue:`27261`)
 - Fix to ensure that tab-completion in an IPython console does not raise
   warnings for deprecated attributes (:issue:`27900`).
+- :func:`qcut` now handles bool ndarray/Series (:issue:`20303`)
```

Comment:
Can you move this to 1.0?  Also, maybe a little more direct to say "boolean input" instead of specifying specific containers.